backend/src/main/java/com/javahelp/backend/data/ISurveyStore.java

```diff
@@ -0,0 +1,40 @@
+package com.javahelp.backend.data;
+
+import com.amazonaws.regions.Regions;
+import com.javahelp.model.survey.Survey;
+
+public interface ISurveyStore {
+    /**
+     * @return Default implementation of {@link ISurveyStore}
+     */
+    static ISurveyStore getDefaultImplementation() {
+        return new DynamoDBSurveyStore("javahelpBackendSurveys", Regions.US_EAST_1);
+    }
+
+    /**
+     * Creates a {@link Survey}
+     *
+     * @param survey {@link Survey} to create
+     * @return {@link Survey} that was created
+     */
+    Survey create(Survey survey);
+
+    /**
+     * @param id {@link String} id of the {@link Survey}
+     * @return {@link Survey} with the specified id
+     */
+    Survey read(String id);
+
+    /**
+     * Deletes the specified {@link Survey}
+     *
+     * @param id {@link String} id of the {@link Survey} to delete
+     */
+    void delete(String id);
+
+    /**
+     * Removes all {@link Survey}s in database.
+     * ONLY use during preliminary testing!
+     */
+    void cleanTable();
```

Comment:
Is the  clean table method  meant to still be here?

Reply:
I think so, at least for now. Otherwise there is no other way to quickly reset the table of Surveys.